commands.js

```diff
@@ -400,6 +446,7 @@ var commands = exports.commands = {
 			return;
 		}
 		if (!this.can('declare')) return false;
+		if (room.isPersonal) return this.errorReply("Personal rooms configuration can't be changed.");
```

Comment:
I'm not sure this is particularly necessary, either. I guess it can't hurt, though...